specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/virtualWan.json

```diff
@@ -1090,6 +1090,145 @@
             }
           }
         }
+      },
+      "put": {
+        "operationId": "HubVirtualNetworkConnections_CreateOrUpdate",
+        "x-ms-examples": {
+          "HubVirtualNetworkConnectionPut": {
+            "$ref": "./examples/HubVirtualNetworkConnectionPut.json"
+          }
+        },
+        "description": "Creates a HubVirtualNetworkConnection resource if it doesn't exist. Updates the HubVirtualNetworkConnection if one exists.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the HubVirtualNetworkConnection."
+          },
+          {
+            "name": "virtualHubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the parent Virtual Hub."
+          },
+          {
+            "name": "connectionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the HubVirtualNetworkConnection."
+          },
```

Comment:
This `connectionName` definition is used multiple times. I'd recommend extracting it to your definitions section, like you've done with `SubscriptionIdParameter`.